test/lit.cfg

```diff
@@ -860,7 +860,7 @@ elif run_os in ['windows-msvc']:
             ('%r -modulewrap -target %s' % (config.swiftc, config.variant_triple))
 
 
-elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'windows-gnu']:
+elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'windows-gnu', 'linux-android']:
```

Comment:
These lit.cfg changes were a hack simply to get lit running natively on Android. I left this in simply to show what was needed: this will need to be changed and done right, after discussion.

I also had to comment out several tests in test/stdlib/InputStream.swift.gyb as they would hang.

Reply:
This seems correct actually.